plugins/woocommerce-blocks/assets/js/base/components/country-input/country-input.tsx

```diff
@@ -4,13 +4,23 @@
 import { useMemo } from '@wordpress/element';
 import { decodeEntities } from '@wordpress/html-entities';
 import clsx from 'clsx';
+import { __ } from '@wordpress/i18n';
+import { useSelect } from '@wordpress/data';
+import { VALIDATION_STORE_KEY } from '@woocommerce/block-data';
+import { ValidationInputError } from '@woocommerce/blocks-components';
 
 /**
  * Internal dependencies
  */
 import './style.scss';
 import type { CountryInputWithCountriesProps } from './CountryInputProps';
-import { Select } from '../select';
+import { Select, SelectOption } from '../select';
+
+const emptyCountryOption: SelectOption = {
+	value: '',
+	label: __( 'Select a country', 'woocommerce' ),
```

Comment:
I assume this is the default for this one? Ideally we should just use `Select a ${label}` which is used in other places.

Reply:
This is fixed now, thanks for spotting this.